Description: build fails on aarch64 with: "Invalid DOS header magic". Project is
not maintained anymore, but this fix was suggested in
https://github.com/ncroxon/gnu-efi/issues/7#issuecomment-2122741592
Upstream: no
diff --git a/Make.rules b/Make.rules
index c8a3ddc..2711cc6 100644
--- a/Make.rules
+++ b/Make.rules
@@ -53,11 +53,6 @@ ifeq ($(ARCH),arm)
   FORMAT = -O binary
 endif
 
-ifeq ($(ARCH),aarch64)
-  LDFLAGS += --defsym=EFI_SUBSYSTEM=0x0a
-  FORMAT = -O binary
-endif
-
 ifeq ($(ARCH),riscv64)
   LDFLAGS += --defsym=EFI_SUBSYSTEM=0x0a
   FORMAT = -O binary
